### ISSUE TITLE
Empty scenes

### ISSUE THREAD

**Issue by vtereshkov** [marekmaskarinec/vaultjam2#4]
If the player moves upwards and crosses the top border of the screen, the new scene is empty.
![image](https://user-images.githubusercontent.com/52492460/113427543-e3923f00-93dd-11eb-87a5-83b35f250421.png)

**Comment by marekmaskarinec** [comment]
Yes all scenes on x=0, y=0 and both are empty. I will fix it, if I have time.
